_src/css/_secdarker.scss

```diff
@@ -0,0 +1,21 @@
+.section--darker {
```

Comment:
@felipprodrigues the way it was before is the correct one. You are styling the same element with a different modifier, so the file should be `.section.scss`